src/session/client-certificate.c

```diff
@@ -41,6 +43,39 @@
 
 #define CLIENT_CERTIFICATE_DIRECTORY   "/run/cockpit/tls/clients"
 
+static int
+open_proc_pid (pid_t pid)
+{
+    char path[100];
+    int r = snprintf (path, sizeof path, "/proc/%lu", (unsigned long) pid);
+    if (r < 0 || r >= sizeof path)
+        errx (EX, "memory error");
+    int fd = open (path, O_DIRECTORY | O_NOFOLLOW | O_PATH | O_CLOEXEC);
+    if (fd < 0)
+      err (EX,  "failed to open %s", path);
```

Comment:
extra space

Reply:
It's called the Hubble tension! 

(Fixed. Apparently my space bar bounces..)